frontends/ol-search-ui/src/components/SearchFilterDrawer.tsx

```diff
@@ -1,7 +1,9 @@
 import React, { useCallback, useState } from "react"
```

Comment:
General changes here:

- replaced `useDeviceCategory` with an `alwaysOpen` prop. The value of `alwaysOpen` (which SearchPage sets based on `useBreakpoint`.
- fixed issue where "close" button was obscured by header
- Some styling changes (see screenshots below) and replaced some divs with actual buttons

**OLD / NEW**
<img width="300" alt="Screen Shot 2022-09-15 at 5 25 55 PM" src="https://user-images.githubusercontent.com/9010790/190511635-5ecab43f-a83f-4b62-ae6a-aae2457f3ecb.png"> / <img width="300" alt="Screen Shot 2022-09-15 at 5 03 25 PM" src="https://user-images.githubusercontent.com/9010790/190511648-3fc63eec-2084-4922-959f-35aceaba0afe.png">